components/scream/tests/generic/infra/CMakeLists.txt

```diff
@@ -4,10 +4,11 @@ set(NEED_LIBS ekat scream_share)
 
 # Add the fake test
 CreateUnitTest(fake_test "fake_test.cpp" "${NEED_LIBS}")
-# CreateUnitTest(resource_spread_thread "resource_spread.cpp" "${NEED_LIBS}"
-#   THREADS 1 4 1
-#   MPI_RANKS 4)
 
-# CreateUnitTest(resource_spread_rank "resource_spread.cpp" "${NEED_LIBS}"
-#   THREADS 4
-#   MPI_RANKS 1 4 1)
+CreateUnitTest(resource_spread_thread "resource_spread.cpp" "${NEED_LIBS}"
+  THREADS 1 4 1
+  MPI_RANKS 4)
+
+CreateUnitTest(resource_spread_rank "resource_spread.cpp" "${NEED_LIBS}"
+  THREADS 4
+  MPI_RANKS 1 4 1)
```

Comment:
Hard coding the max threads/ranks number might give headaches to some extra diligent developer who runs these tests on a small machine (perhaps b/c they suspect something messed up spreading). These two sets of tests have a max of 16 threads needed. Although that's small for clusters, some very diligent dev might run them on a workstation with 10 cores, and get config errors. We could use `SCREAM_TEST_MAX_RANKS` and `SCREAM_TEST_MAX_THREADS`, to give them the ability to stay within machine bounds.